TROUBLESHOOTING.md

```diff
@@ -1,60 +1,63 @@
 # Troubleshooting
 
-* [Tools](#tools)
-* [Installation](#installation)
-  * [InternalError when calling webhook: context deadline exceeded](#internalerror-when-calling-webhook-context-deadline-exceeded)
-  * [Invalid header field value](#invalid-header-field-value)
-  * [Helm chart install failure: certificate signed by unknown authority](#helm-chart-install-failure-certificate-signed-by-unknown-authority)
-* [Operations](#operations)
-  * [Stuck runner kind or backing pod](#stuck-runner-kind-or-backing-pod)
-  * [Delay in jobs being allocated to runners](#delay-in-jobs-being-allocated-to-runners)
-  * [Runner coming up before network available](#runner-coming-up-before-network-available)
-  * [Outgoing network action hangs indefinitely](#outgoing-network-action-hangs-indefinitely)
-  * [Unable to scale to zero with TotalNumberOfQueuedAndInProgressWorkflowRuns](#unable-to-scale-to-zero-with-totalnumberofqueuedandinprogressworkflowruns)
-  * [Slow / failure to boot dind sidecar (default runner)](#slow--failure-to-boot-dind-sidecar-default-runner)
+- [Troubleshooting](#troubleshooting)
+  - [Tools](#tools)
+  - [Installation](#installation)
+    - [InternalError when calling webhook: context deadline exceeded](#internalerror-when-calling-webhook-context-deadline-exceeded)
+    - [Invalid header field value](#invalid-header-field-value)
+    - [Helm chart install failure: certificate signed by unknown authority](#helm-chart-install-failure-certificate-signed-by-unknown-authority)
+  - [Operations](#operations)
+    - [Stuck runner kind or backing pod](#stuck-runner-kind-or-backing-pod)
+    - [Delay in jobs being allocated to runners](#delay-in-jobs-being-allocated-to-runners)
+    - [Runner coming up before network available](#runner-coming-up-before-network-available)
+    - [Outgoing network action hangs indefinitely](#outgoing-network-action-hangs-indefinitely)
+    - [Unable to scale to zero with TotalNumberOfQueuedAndInProgressWorkflowRuns](#unable-to-scale-to-zero-with-totalnumberofqueuedandinprogressworkflowruns)
+    - [Slow / failure to boot dind sidecar (default runner)](#slow--failure-to-boot-dind-sidecar-default-runner)
```

Comment:
I'm not sure what changed here besides adding `Troubleshooting` and updating the hierarchy. I don't see the purpose of having the h1 added to the TOC here.